spec/unit/auto_link_spec.rb

```diff
@@ -14,6 +14,10 @@ def admin_post_path(post)
     "/admin/posts/#{post.id}"
   end
 
+  def authorized?(action, subject)
```

Comment:
Unused method argument - `action`. If it's necessary, use `_` or `_action` as an argument name to indicate that it won't be used. You can also write as `authorized?(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `subject`. If it's necessary, use `_` or `_subject` as an argument name to indicate that it won't be used. You can also write as `authorized?(*)` if you want the method to accept any arguments but don't care about them.